app/models/application_record.rb

```diff
@@ -58,23 +63,24 @@ def pending?
     !reviewed
   end
 
-  def self.gender_eeo_data(start_date, end_date)
+  def self.gender_eeo_data(start_date, end_date, department_id)
     gender_records = between(start_date, end_date).with_gender
+                     .in_department(department_id)
     all_genders = GENDER_OPTIONS | gender_records.pluck(:gender)
     all_genders.map do |option|
       [option, gender_records.where(gender: option).count]
     end
   end
 
-  def self.eeo_data(start_date, end_date)
+  def self.eeo_data(start_date, end_date, department_id)
     records = {}
-    records[:all] = between(start_date, end_date)
+    records[:all] = between(start_date, end_date).in_department(department_id)
```

Comment:
I'd prefer to leave off the final parentheses here - I think it reads better as a prepositional phrase that way.